app/src/components/market/sections/market_create/steps/outcomes/index.tsx

```diff
@@ -68,7 +68,7 @@ const CustomButtonCircle = styled(ButtonCircle)`
 
 const CustomButtonCircleAddReadyCSS = css`
   background-color: ${props => props.theme.colors.primary};
-  border-color: ${props => props.theme.colors.primary};
+  border-color: ${props => props.theme.colors.primary} !important;
```

Comment:
We shouldn't use `!important` unless completely necessary an unavoidable. This should do the trick and create all the necessary CSS classes:

```suggestion
  & {
    background-color: ${props => props.theme.colors.primary};
    border-color: ${props => props.theme.colors.primary};

    path {
      fill: #fff;
    }
  }
```

Reply:
@gabitoesmiapodo indeed this does the trick, wasn't aware of this – whats actually the trick here?

Reply:
It seems it creates a new class for this particular case instead of cascading classes as it did before.